python/ray/_private/runtime_env/cgroup.py

```diff
@@ -0,0 +1,19 @@
+import logging
+from typing import Optional
+
+from ray._private.runtime_env import RuntimeEnvContext
+
+default_logger = logging.getLogger(__name__)
+
+
+class CgroupManager:
+    def __init__(self):
+        pass
+
+    def setup(self,
+              allocated_resource: dict,
```

Comment:
Can you make this a kwarg so it's optional given that it won't be used by most plugins? Also, please add type annotattions for the dictionary (keys and values)